docs/DeveloperGuide.md

```diff
@@ -480,8 +480,38 @@ The following activity diagram shows the logic of the `help` command.
 
 **Overview:**
 
+The `add-t` command is used to add additional tags to an existing `Person` in ConnectUS specified by the user.
+
+The format for the `add-t` command can be found [here](https://ay2223s2-cs2103t-w15-1.github.io/tp/UserGuide.html#46-adding-additional-tags-to-a-contact-add-t).
+
+<div style="page-break-after: always"></div>
+
 **Feature Details:**
 
+1. The user specifies a person index that represents a `Person` to be edited, followed by the tag to be added.
+2. If a negative or zero index is provided, an error is thrown. The user is prompted to re-enter the command correctly.
+3. If the index is not in valid range of the contact list provided, an error is thrown. The user is prompted to re-enter the command correctly.
```

Comment:
Perhaps it would also be good to add description for this code from AddTagToPersonCommand:

```java
if (editedPerson.getMajors().size() > Major.MAX_MAJOR_COUNT) {
            throw new CommandException(MESSAGE_PERSON_TOO_MANY_MAJORS);
        }
```

Reply:
very minor (not good representation), resolving